configuration/config2/SchemaTypeStringLimits.cpp

```diff
@@ -35,8 +35,8 @@ bool SchemaTypeStringLimits::doValueTest(const std::string &testValue) const
     // test patterns
     for (auto it = m_patterns.begin(); it != m_patterns.end() && isValid; ++it)
     {
-        std::regex expr ((*it).c_str());
-        isValid = std::regex_match(testValue, expr);
+        static const boost::regex e(*it);
```

Comment:
not entirely sure why e is declared as static, also the var name could be more descriptive.

Reply:
Well, because it was pasted code and I should have removed it. Also changed name to expr